modules/hub/src/ApiServer.ts

```diff
@@ -176,7 +172,7 @@ export class ApiServer {
   ) {
     const roles = await this.authHandler.rolesFor(req)
     req.session!.roles = new Set(roles)
-    const allowed = true // await this.authHandler.isAuthorized(req)
+    const allowed = await this.authHandler.isAuthorized(req)
```

Comment:
This was commented out earlier, hope reactivating it doesn't cause any issues that I don't know about..